Artifacts/windows-clone-git-repo/GitEnlister.ps1

```diff
@@ -76,6 +76,9 @@ $ScriptRoot = $args[4]
 # Note: Because the $ErrorActionPreference is "Stop", this script will stop on first failure.  
 $ErrorActionPreference = "stop"
 
+# Ensure that current process can run scripts. 
+Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process -Force 

```

Comment:
Is "-ExecutionPolicy bypass" needed here? Since the invoked script (Artifacts/windows-chocolatey/startChocolatey.ps1) seems to set it anyways.